api/v1/controllers/lenses.js

```diff
@@ -410,18 +411,46 @@ module.exports = {
 
       updateLensDetails(seqObj);
       const assocToCreate = u.includeAssocToCreate(seqObj, helper);
-      helper.model.create(seqObj, assocToCreate)
-      .then((o) => {
-        resultObj.dbTime = new Date() - resultObj.reqStartTime;
-        delete o.dataValues.library;
-        u.logAPI(req, resultObj, o.dataValues);
-        res.status(httpStatus.CREATED).json(
-          u.responsify(o, helper, req.method)
-        );
-      })
-      .catch((err) => {
-        u.handleError(next, err, helper.modelName);
-      });
+      if (!featureToggles.isFeatureEnabled('returnCreatedBy')) {
```

Comment:
@pallavi2209 where do you get these errors? Running npm run jscs-enforce does not produce the same results.